### DISCUSSION TITLE
📸  Built with FlutterFire showcase

### DISCUSSION THREAD

**Discussion by Ehesp** [firebase/flutterfire#3366] [Show and tell]
If you've built something using FlutterFire and would like to share it with everyone please add a comment below 🔥

---

- **Application / Product Name:**
- **Play Store URL**:
- **App Store URL**:
- **Web App URL**: 
- **Firebase Plugins** (e.g: firestore, analytics):
- **Additional details**:

**Comment by disparta** [comment]
- **Application Name**: notes

- **Play Store URL**: https://play.google.com/store/apps/details?id=tech.notes.android

- **App Store URL**: https://apps.apple.com/app/id1528770116

- **Web App URL**: Not yet available, some Firebase Plugins do not support web, but here is our site: https://notestech.co.za.

- **Firebase Plugins**: Auth, Analytics, Crashlytics, Performance, Firestore, Dynamic Links, Storage, Remote-Config.

- **Additional Details**: Helps students share study notes and summaries with other students. Subscription-based and creators earn based on the usage of their content, subscriptions are currently disabled because we are looking for students to upload their notes/summaries.

**Comment by IchordeDionysos** [comment]
- Application / Product Name: simpleclub
- Play Store URL: https://play.google.com/store/apps/details?id=com.simpleclub.android
- App Store URL: https://apps.apple.com/de/app/simpleclub-die-lernapp/id1089299986
- Web App URL: https://app.simpleclub.com
- Firebase Plugins: Analytics, Authentication, Firestore, Storage, Remote Config, Dynamic Links, Crashlytics, Messaging, In-App Messaging, Performance
- Additional details: simpleclub is the most popular student education app in Germany, we produce high quality, engaging educational videos, exercises and interactive summaries all powered and made possible by Flutter and Firebase. Also thanks to the [Rive](https://rive.app) team for making these interactive summaries so easy 💪
We even empower our content creators to build these exercises and summaries in an internal tool also built in Flutter. 😎

**Comment by nilsreichardt** [comment]
* Application / Product Name: Sharezone
* Play Store URL: https://sharezone.net/android
* (iOS) App Store URL: https://sharezone.net/ios
* (macOS) App Store URL: https://sharezone.net/macos
* Web App URL: https://sharezone.net/web
* Firebase Plugins: Analytics, Authentication, Firestore, Storage, Remote Config, Dynamic Links, Crashlytics, Messaging, Performance, Cloud Functions
* Additional details: Sharezone is a collaborative organization app with already more than 200,000 registered pupils and a rating of 4.7 in the AppStore. It takes the struggle of organizing the school day by allowing pupils to organize themselves together (Note: Sharezone is currently only in German available). We are building Sharezone with Flutter + Firebase (Android, iOS, Web & macOS). For us is Flutter & Firebase the proof how easy it is to build a beautiful, fast and secure app. We had no experience before Sharezone 😅 I mean if two young student with no experience can build an app thanks to Flutter & Firebase, who can't?
